packages/admin-ui/src/form-group/form-group.tsx

```diff
@@ -0,0 +1,83 @@
+import type { ReactNode } from 'react'
+import React, { Fragment } from 'react'
+import { createComponent } from '@vtex/admin-ui-react'
+
+import { messages } from './form-group.i18n'
+import { Text } from '../components/Text'
+import { Stack } from '../stack'
+import { Message } from './message'
+import { useMessageFormatter } from '../i18n'
+
+export const FormGroup = createComponent<'div', any>((props) => {
+  const {
+    label,
+    labelAs = 'span',
+    direction = 'row',
+    children,
+    helpText,
+    error = false,
+    errorText,
+    state = null,
+    optional = false,
+    componentGroup: ComponentGroup = Fragment,
+  } = props
+
+  const formatMessage = useMessageFormatter(messages.formGroup)
+
+  return (
+    <Stack space="$l">
+      <Text as={labelAs} variant="detail" tone="secondary">
```

Comment:
This was made only because radio label which will be used "legend". We could remove it and let Radio Group to handle its label for itself. This prop as is not something i was happy about it, but i was worried how Radio Group would handle this... Do you have suggestions?

Reply:
Yeah, I think that my suggestion for the `FormControl` to be composable would solve this.